kernel/thinkpad.txt

```diff
@@ -0,0 +1,28 @@
+Kernel configuration for IBM ThinkPad
+________________________________________________________________________________
+
+
+X230
+________________________________________________________________________________
+
+A defconfig is sufficent for the machine to boot. Sound and WIFI must be manu-
+ally enabled. A firmware blob for INTEL Centrino Advanced-N 6205 is necessary.
+Thinkpad specific ACPI options _can_ be enabled.
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make defconfig                                                           |
+|   $ cp iwlwifi-6000g2a-6.ucode /usr/lib/firmware                             |
+|                                                                              |
++------------------------------------------------------------------------------+
+
++------------------------------------------------------------------------------+
+| .config                                                                      |
++------------------------------------------------------------------------------+
+|                                                                              |
+|   CONFIG_IWLWIFI=m                                                           |
+|   CONFIG_IWLDVM=m                                                            |
```

Comment:
We should recommend `=y` and baking in the firmware. See: https://k1ss.org/wiki/kernel/firmware